client/src/utils.ts

```diff
@@ -199,7 +200,7 @@ class Utils {
 		return error;
 	}
 
-	public getTestTitlePath(contextOrSuite: Mocha.Context | Mocha.Suite) {
+	public getTestTitlePath(contextOrSuite: Mocha.Context | Mocha.Suite, sterilize = true) {
```

Comment:
I feel like `sanitize` is a better name for this? `sterilize` feels a little off. haha